README.md

```diff
@@ -2,6 +2,28 @@
 
 This content repository allows easy content management via a headless CMS, using [wagtail](https://wagtail.io/) that is meant for web and various messaging platforms.
 
+## Features
+
+- **Multi-channel support:** Currently supports web, WhatsApp, Messenger, and Viber.
+- **Ordered content sets:** You can take individual pieces of content and group them in a set, in a specific order. They can also be restricted to users with specific profile field values.
+- **Stage based messaging:** Ordered content sets can be expanded to include a relative time (eg. 23 days before x), to create a timing-based push-message journey.
+- **Import and export:** Content and ordered content sets can be exported and imported in CSV and Excel formats
+- **Page views:** Content page views for each channel can be stored, with reporting in the admin UI. Custom fields for each page view can also be stored (eg. user ID)
+- **Rich media support:** Can store images for Messenger and Viber, and images, documents, audio and video for WhatsApp.
+- **Custom branding:** Can configure the title, login and welcome messages, logo, and favicon for the content admin UI
+- **Variation messaging:** Support for variations on WhatsApp messages for different user profiles fields: age, gender, and relationship status.
+- **Content tree:** Content is stored in a tree format, allowing arbitrary-depth nested menu structures
+- **Tags:** Content can be tagged to group certain kinds of content together
+- **Triggers:** Triggers can be added to content, to specify custom triggers for when a piece of content should be sent
+- **Quick replies:** A list of quick replies/buttons can be added to the content
+- **Related pages:** Each piece of content can be linked to other content pages that are related to it, that can be suggested to users.
+- **WhatsApp templates:** WhatsApp template messages can be stored, with the option of also submitting them to the WhatsApp Business API for approval
+- **Page ratings:** Content pages can be rated by the user as helpful or not helpful, with text comments.
+- **API authentication:** The API requires authentication to access. This can be through an authentication token, basic authentication, or session cookie.
+- **Suggested content:** Return a random sample of child pages from a supplied list of parent pages.
+- **Search:** Find content pages through a search query.
+
```

Comment:
We can add 

- Internationlisation, we allow for content to be translated
- Reports on page views and stale content
- Workflow Management